src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java

```diff
@@ -82,8 +88,7 @@ public CommandManagerWindow(DriverCommandManager commandManager) {
 	}
 
 	private void runCommand() {
-		commandManager.clearCurrentCommand();
-		updateCurrentCommandField();
+		commandManager.runCurrentCommand().execute(driverManager.getCurrentDriver());;
```

Comment:
For now it is ok, but for future work: It would be better not to introduce the direct dependency to DriverManager (this is a view).